### PR TITLE
Task: Set up automatic deployment with Travis

### PR DESCRIPTION
**Pull request by paulathevalley**
Hook up travis to re-deploy the docs site with every push to `master`.

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -1,30 +1,37 @@
...
   - npm install -g npm@3
 
 script:
   - npm test
+  - NODE_ENV=production node_modules/.bin/builder run build-static
 
-  # Prune deps to just production and ensure we can build the site
-  - npm prune --production
-  - npm run build-static
+after_success:
```
Comment (paulathevalley) [inline .travis.yml]: Note to self: change this to deploy only on `master`, e.g. https://github.com/FormidableLabs/victory-docs/blob/master/.travis.yml#L34

File: deploy.sh [hunk 2]
```diff
@@ -0,0 +1,10 @@
+#!/bin/sh
+
+# make sure key is permissive, but not too permissive
+chmod 600 deploy_static.pem
+# clean out any existing staging folder but make sure it exists
+ssh -i deploy_static.pem formidable@192.241.218.94 "rm -rf static/builder-staging && mkdir -p static/builder-staging"
+# copy the spectacle build to the staging arena; if this fails, site is still OK
+scp -i deploy_static.pem -rp ./build/* formidable@192.241.218.94:/home/formidable/static/spectacle-staging
```
Comment (coopy) [inline deploy.sh]: This line says `spectacle`, and so does the comment below.

Comment (paulathevalley) [inline deploy.sh]: :facepalm:

**Comment by coopy** [comment]
🚀 !

**Pull request merged**